webpack/scenes/ModuleStreams/Details/__tests__/__snapshots__/ModuleStreamDetailsActions.test.js.snap

```diff
@@ -134,11 +134,3 @@ Array [
   ],
 ]
 `;
-
-exports[`Module stream details actions should load module stream details on success: API get call 1`] = `
-Array [
-  Array [
-    "/module_streams/1",
-  ],
-]
-`;
```

Comment:
Why is this and some other snapshots removed?

Reply:
Just some snapshot confusion.  hopefully fixing now